modules/browser/BrowserCapabilities.js

```diff
@@ -233,8 +233,7 @@ export default class BrowserCapabilities extends BrowserDetection {
      * @returns {boolean}
      */
     supportsScalabilityModeAPI() {
-        return (this.isChromiumBased() && this.isEngineVersionGreaterThan(112))
-            || (this.isFirefox() && this.isVersionGreaterThan(135));
+        return this.isChromiumBased() && this.isEngineVersionGreaterThan(112);
```

Comment:
We will have to bring the check back in for the reason mentioned above where we default to Simulcast for Firefox until the Firefox bug is fixed.

Reply:
Do you mean we should bring it back once the bug is fixed? Meaning no actionable item is related to this comment? 


Also just FYI: There actually isn't a FF bug. I think they just don't support scalabilityModeEnabled prop at all. They never implemented it. At least this is how I understand the current situation with FF.